apps/mdn/mdn-aws/docs/mdn-dd-redis.md

```diff
@@ -0,0 +1,98 @@
+### How does this work?
+
+#### Secrets
+
+The secrets titled `datadog-secrets` has 2 key/value pairs: `API_KEY` and `REDIS_CONFIG_FILE`. 
+
+`API_KEY` is a simple base64 encoded value:
+
+```
+echo -n "some_api_key_value" | base64
+```
+
+`REDIS_CONFIG_FILE` is the base64 encoded contents of the [Datadog Redis configuration](https://docs.datadoghq.com/integrations/redisdb/):
```

Comment:
This wasn't what I was expecting. I thought we'd have a ``redisdb.yml.j2`` that was combined with secrets. But, if it works, and is the way preferred by the Datadog image, then 👍

Reply:
we have to specify some secret values inside the redisdb.yaml file, and the DD agent Docker image needs the files upon startup to enable the Redis collector. The easiest way to pass in a file with secret data is just by mounting it as a secret. The private MDN repo contains the "rendered" datadog-secrets file, including the contents of redisdb.yaml in base64 format.